test/test_retriever.py

```diff
@@ -52,6 +53,12 @@
 tar_gz_url = os.path.normpath(achive_url.format(file_path='sample_tar.tar.gz'))
 gz_url = os.path.normpath(achive_url.format(file_path='sample.gz'))
 
+kaggle_datasets = [
+    # test_name, data_source, dataset_name, dataset_name, repath, expected
```

Comment:
`dataset_name, dataset_name`
We should change the names here. Both can't be the same.